packages/aeproject-cli/aeproject-fire-editor/fire-editor.js

```diff
@@ -29,6 +29,11 @@ const moduleName = constants.MODULE_NAME;
 const isWindowsPlatform = process.platform === 'win32';
 
 const run = async (options) => {
+    let result = await isNodeVersionSupported();
+    
+    if (!(result >= 0)) {
```

Comment:
result < 0 ?

Reply:
1 - greater
0 - equal
-1 lower
if i am checking the positive scenario all the try-catch should be enclosed inside the if